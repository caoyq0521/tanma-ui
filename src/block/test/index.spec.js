import { mount } from '@vue/test-utils';
import TmBlock from '../block.vue';

describe('tmBlock', () => {
  it('render block', () => {
    const wrapper = mount(TmBlock);
    expect(wrapper).toMatchSnapshot();
  });

  it('title', () => {
    const wrapper = mount({
      template: `<tm-block title="测试标题">测试内容</tm-block>`
    });

    expect(wrapper.find('.tm-block__title').text()).toContain('测试标题');

    expect(wrapper.find('.tm-block__content').text()).toContain('测试内容');
  });
})
  