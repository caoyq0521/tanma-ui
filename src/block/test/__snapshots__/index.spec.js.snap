// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tmBlock render block 1`] = `
<div class="tm-block">
  <div class="tm-block__title"></div>
  <div class="tm-block__content"></div>
</div>
`;
